src/index.html

```diff
@@ -11,7 +11,192 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="myFieldset">
+        <legend>Personal information</legend>
+
+        <div>
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            id="surname"
+            name="surname"
+            autocomplete="off"
+            minlength="5"
+          >
+        </div>
+
+        <div class="inputs">
+          <label for="name">Name:</label>
+          <input
+            type="text"
+            id="name"
+            name="name"
+            autocomplete="off"
+            minlength="4"
+          >
+        </div>
+
+
+        <div class="inputs">
```

Comment:
```suggestion
        </div>

        <div class="inputs">
```

Too many blank lines, fix everywhere